packages/accordions/src/elements/stepper/components/Step.spec.tsx

```diff
@@ -46,4 +46,38 @@ describe('Step', () => {
 
     expect(queryAllByTestId('step-line')).toHaveLength(0);
   });
+
+  it('does not change step number when running outside strict mode', async () => {
```

Comment:
I think these tests are accurate, but should we also be asserting that the steps are in the correct order as well?

Reply:
I'm not sure what the best way to do this is, so I've used `findAllByText`, and then asserted the order of elements in the array

Reply:
LGTM 👍